internal/cli/server/flags.go

```diff
@@ -56,7 +56,7 @@ func (c *Command) Flags() *flagset.Flagset {
 		Default: c.cliConfig.GcMode,
 	})
 	f.MapStringFlag(&flagset.MapStringFlag{
-		Name:  "requiredblocks",
+		Name:  "eth.requiredblocks",
 		Usage: "Comma separated block number-to-hash mappings to enforce (<number>=<hash>)",
```

Comment:
Can we write the usage similar to what is present in geth: `Comma separated block number-to-hash mappings to require for peering (<number>=<hash>)`

Reply:
Yes, sure.